### PR TITLE
Add multi input

### LINKED ISSUE #51: Installing multiple apps at once
When I try to install multiple apps using something like `sudo pacstall -I neofetch micro other-app` pacstall only installs the first in the list. There are no warnings or messages, it just only installs the first, in this case neofetch. This is unlike the behavior of every other package manager that I have used and can be annoying. Adding some way of installing multiple apps would be very helpful. 
Steps to reproduce:
1. Install pacstall.
2. Run `sudo pacstall -I` with more than one argument.
This will only install the first item in the list of arguments.

Thank you.

### PR DESCRIPTION
**Pull request by Henryws**
Closes #51

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
I get this error
```
/usr/bin/pacstall: line 219: syntax error near unexpected token `;;'
/usr/bin/pacstall: line 219: `      ;;'
```

**Comment by Henryws** [comment]
It works for me

File: pacstall [hunk 1]
```diff
@@ -196,7 +196,9 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
         fancy_message error "Run as root"
         exit 1
       fi
-      PACKAGE=$2
+      PACKAGE="$@"
+      PACKAGE=$(echo $PACKAGE | cut -dI -f2- | sed 's/ //')
+      for PACKAGE in $PACKAGE; do
```
Comment (wizard-28) [inline pacstall:201]: No `done` after `do`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: pacstall (hunk 1 above)
Comment (Henryws) [reply pacstall:201]: ?

Comment (Henryws) [reply pacstall:201]: for i in {1,2,3}; do echo $i ; done

Comment (wizard-28) [reply pacstall:201]: There is no `done` statement to close this `do` block. I mean you opened the `for` loop but you didn't end it.

Comment (Henryws) [reply pacstall:201]: At the very end, I added `done` before the exit

Comment (wizard-28) [reply pacstall:201]: Line number?

Comment (Henryws) [reply pacstall:201]: 237

Comment (wizard-28) [reply pacstall:201]: 237 is under `-Il` case I am talking about the `for` loop of `-I` case (line number 201)

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Its trying to invalid `optdepends`
```
~
❯ sudo pacstall -I neofetch webapp-manager
neofetch.pacscript                100%[=============================================================>]     652  --.-KB/s    in 0s
Do you want to view the pacscript first [y/N]
Do you want to edit the pacscript [y/N]
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
7.1.0.zip                             [  <=>                                                         ]  95.00K   223KB/s    in 0.4s
[+] INFO: Installing
[+] INFO: Package has some optional dependencies that can enhance its functionalities
Optional dependencies:
    w3m-img: image preview
    libsixel-dev: image preview
    catimg: image preview
    jp2a: image preview
    caca-utils: image preview
    libcaca-dev: image preview
Do you want to install them [Y/n] n
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
webapp-manager.pacscript          100%[=============================================================>]     904  --.-KB/s    in 0s
Do you want to view the pacscript first [y/N] y
Do you want to edit the pacscript [y/N]
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: webapp-manager requires gettext to install
Do you want to remove them after installing webapp-manager [y/N]
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
1.1.2.tar.gz                          [  <=>                                                         ]  79.67K   242KB/s    in 0.3s
[+] INFO: Installing
[+] INFO: Package has some optional dependencies that can enhance its functionalities
Optional dependencies:
    w3m-img: image preview
    libsixel-dev: image preview
    catimg: image preview
    jp2a: image preview
    caca-utils: image preview
    libcaca-dev: image preview
Do you want to install them [Y/n] n
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
```

Here its trying to install neofetch's `optdepends` in case of `webapp-manager`

**Comment by Henryws** [comment]
I think it might be because pacstall isn't unsetting variables after installation so if neofetch has optional dependencies, it carry's over to webapp-manager, because webapp-manager has no optional dependencies to overwrite

**Comment by Henryws** [comment]
So after installation, pacstall needs to run `unset var` on every pacstall variable

**Comment by Henryws** [comment]
`unset {name,version,url,build_depends,depends,breaks,replace,description,hash,removescript,optdepends,ppa,maintainer,pacdeps,patch,prepare,build,install}`

**Comment by wizard-28** [comment]
Its not even trying to install the optional dependencies of any of the pacscripts
```
❯ sudo pacstall -P -I neofetch webapp-manager
[*] WARNING: Prompts are disabled
neofetch.pacscript                100%[=============================================================>]     652  --.-KB/s    in 0s
Do you want to view the pacscript first [y/N] N
Do you want to edit the pacscript [y/N] N
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
7.1.0.zip                             [   <=>                                                        ]  95.00K   193KB/s    in 0.5s
[+] INFO: Installing
[+] INFO: Package has some optional dependencies that can enhance its functionalities
Optional dependencies:
    w3m-img: image preview
    libsixel-dev: image preview
    catimg: image preview
    jp2a: image preview
    caca-utils: image preview
    libcaca-dev: image preview
Do you want to install them [Y/n] Y
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
webapp-manager.pacscript          100%[=============================================================>]     904  --.-KB/s    in 0s
Do you want to view the pacscript first [y/N] N
Do you want to edit the pacscript [y/N] N
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: webapp-manager requires gettext to install
Do you want to remove them after installing webapp-manager [y/N] N
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
1.1.2.tar.gz                          [ <=>                                                          ]  79.67K  --.-KB/s    in 0.1s
[+] INFO: Installing
[+] INFO: Package has some optional dependencies that can enhance its functionalities
Optional dependencies:
    w3m-img: image preview
    libsixel-dev: image preview
    catimg: image preview
    jp2a: image preview
    caca-utils: image preview
    libcaca-dev: image preview
Do you want to install them [Y/n] Y
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
```

None of the optional dependencies were installed on my system

**Comment by Henryws** [comment]
```bash
    if ask "Do you want to install them" Y; then
        for items in "${optdepends[*]}"; do
            printf "%s\n" "$items" | cut -d: -f1 | tr '\n' ' ' | cut -d% -f1 >> /tmp/pacstall-optdepends
            sudo apt-get install -y -qq -o=Dpkg::Use-Pty=0 $(cat /tmp/pacstall-optdepends)
        done
    fi
```

**Comment by wizard-28** [comment]
> ```shell
>     if ask "Do you want to install them" Y; then
>         for items in "${optdepends[*]}"; do
>             printf "%s\n" "$items" | cut -d: -f1 | tr '\n' ' ' | cut -d% -f1 >> /tmp/pacstall-optdepends
>             sudo apt-get install -y -qq -o=Dpkg::Use-Pty=0 $(cat /tmp/pacstall-optdepends)
>         done
>     fi
> ```

So... I guess we found a new bug in optional dependencies...

**Comment by Henryws** [comment]
```bash
Optional dependencies:
    w3m-img: image preview
    libsixel-dev: image preview
    catimg: image preview
    jp2a: image preview
    caca-utils: image preview
    libcaca-dev: image preview
Do you want to install them [Y/n] y
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
➜  ~ cat /tmp/pacstall-optdepends
w3m-img
➜  ~
```

**Comment by Henryws** [comment]
```bash
printf "%s\n" "${optdepends[@]}" | cut -f1 -d":" | tr '\n' ' ' >> /tmp/pacstall-optdepends
sudo apt-get install -y -qq << /tmp/pacstall-optdepends
```

**Comment by wizard-28** [comment]
> ```shell
> printf "%s\n" "${optdepends[@]}" | cut -f1 -d":" | tr '\n' ' ' >> /tmp/pacstall-optdepends
> sudo apt-get install -y -qq $(cat /tmp/pacstall-optdepends)
> ```

Make a separate PR fixing this, this code works

**Comment by Henryws** [comment]
Merge?

**Comment by wizard-28** [comment]
> `unset {name,version,url,build_depends,depends,breaks,replace,description,hash,removescript,optdepends,ppa,maintainer,pacdeps,patch,prepare,build,install} >/dev/null`

Do this

**Comment by wizard-28** [comment]
And since we are adding multiinput to `-I` flag, I think we should do the same for `-R`, `-Il` and others

**Comment by Henryws** [comment]
One thing I noticed is that this might deprecate long flag names like `--install` over `-I` because it wants to cut a letter from the flag to make `-I pkg1 pkg2` look like `pkg1 pkg2`. I think we might need a better way of doing this

**Review by wizard-28 — CHANGES_REQUESTED** [review]
`-R` is broken.
Its doing nothing on running

**Comment by Henryws** [comment]
Ugh

**Comment by Henryws** [comment]
Maybe we should close the pr and come back to it another time

**Comment by Henryws** [comment]
Did we merge into develop yet?

**Comment by Henryws** [comment]
Nevermind

**Pull request closed without merging**